src/neuro_relay/__main__.py

```
@@ -282,13 +332,16 @@ def _stop(*_):
     except NotImplementedError:
         # Windows fallback: use add_reader on stdin
         import threading
+
         def wait_for_ctrl_c():
```

Ruff (0.14.8)

Missing return type annotation for private function `wait_for_ctrl_c`

Add return type annotation: `None`

(ANN202)